ledger-core/virtual/object/object.go

```diff
@@ -42,6 +42,8 @@ const (
 
 const waitStatePulsePercent = 10
 
+const UnorderedMaxParallelism = 30
```

Comment:
Is it a temporary limit until PLAT-429 (hierarchical semaphores) is done?

Reply:
this is what we had in the code, I just moved it into constant.